Sources/SwiftSyntax/RawSyntax.swift

```diff
@@ -1156,96 +345,186 @@ final class RawSyntax: ManagedBuffer<RawSyntaxBase, RawSyntaxDataElement> {
   ///   - layout: The children of this node.
   /// - Returns: A new RawSyntax `.node` with the provided kind and layout, with
   ///            `.missing` source presence.
+  // @available(*, deprecated, message: "use 'makeEmptyLayout()' with SyntaxArena")
```

Comment:
It's intentional I commented this out. Uncommenting this causes excess number of `warning` in the current `SyntaxFactory` etc. that prevents us from seeing the actual errors/warnings.

Since `SyntaxArena.default` from 5 places, I left it `deprecated`

Reply:
OK, sounds fair to me. But I think to be consistent, we should also uncomment the deprecation of `SyntaxArena.default`. That way SwiftSyntax still builds without warnings.

Reply:
I commented-out the deprecation of `SyntaxArena.default`.